### ISSUE TITLE
Text area focus color does not pass contrast ratio for a11y

### ISSUE THREAD

**Issue by rbirkgit** [vmware/clarity#5098]
## Describe the bug

We have an a11y bug reported on our product that the text area focus color does not have enough contrast (2.0 where 3.0 is needed).

The texarea focus color in Clarity v2-v4 is hard coded to #69C0E2. I am wondering why the focus color is brighter and different from the input field (which uses #0072A3). Would make sense to have the text area using the same color as all other controls, e.g. input, checkbox, radio, select.

Further on, I think we should have fixed focus color for buttons too. We have multiple a11y issues reported where users have have hard time seeing button focus (especially primary buttons).

## How to reproduce

Focus any text area
Focus buttons

## Expected behavior

Same focus color as input

## Versions

**App**

- Clarity: 2, 3 and 4

**Device:**

- Browser [chrome]
- Version [85]

**Comment by colinreedmiller** [comment]
@chlane lets review this together for compliance and strategy.

**Comment by chlane** [comment]
See our recommendations for visual focus indication for all elements that get focus at https://confluence.eng.vmware.com/display/A11Y/Visual+Focus+Indication

**Comment by mathisscott** [comment]
@chlane 
While not visible to the open-source community, I've noted deficiencies in that recommendation. We cannot move forward with that. 

Once the recommendation is vetted across all use cases and signed off on by design/ux we should be able to implement it. Until then, we're stuck.

**Comment by colinreedmiller** [comment]
My intent in suggesting  we need to review for strategy is as Scott says 'vetted across all use cases' - we have several approaches across the library and given the current color system work this is an excellent opportunity to develop a comprehensive strategy.

**Comment by gnomeontherun** [comment]
Clarity cares deeply about accessibility, and we work with our accessibility team to identify and triage issues. We have an internal tracking system to identify and verify accessibility issues, which has captured this already. Some issues with Clarity Angular are difficult to solve without causing side effects or breaking changes, and we have to balance concerns with fixing issues and breaking changes. As we build out Clarity Core, we are able to address these root cause accessibility issues and recommend adopting the Clarity Core implementations as they become available. We are closing this issue as we have the issues tracked internally. Please follow our development and releases to see when we release relevant Clarity Core components to make this possible.
